src/database/migrations/20200110164415-add-user-token.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
You don't have to keep the token in users table as now we are using a different approach